plugins/woocommerce/changelog/update-unify-jest-27

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Update `jest@^26.5.1` from `^25`.
```

Comment:
This description doesn't seem to match the changelog name is that intentional? Shouldn't it mention jest 27 somewhere in the description?